compiler/circle-opselector-test/README.md

```diff
@@ -0,0 +1,52 @@
+# Circle-OpSelector-test
```

Comment:
`Circle-OpSelector-test` vs `circle-opselector-test`

Reply:
We'll use `circle-opselector-test`.